cves/kernel/CVE-2022-3108.yml

```diff
@@ -55,7 +55,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  It seems that the this is an error in the AMD gpu Kernal Fusion Driver which I admittedly know little about other than what I have quickly researched
+  This vulnerability has to do with the Direct Rendering Manager which is a kernal subsystem used to interface with video cards. The actual vulnerability itself
+  is an unchecked return that could result in a null pointer which would be an effective denial of service attack as it would make it so that the user programs may not 
+  be able to communicate with the gpu.
```

Comment:
Dont use "it seems" and remove the part about not knowing much about hte kernel fusion driver, it takes away from your description.

Reply:
Ok will do.